spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/serializer/AbstractByteArraySerializer.java

```diff
@@ -39,7 +39,13 @@
 		Deserializer<byte[]>,
 		ApplicationEventPublisherAware {
 
-	protected int maxMessageSize = 2048;
+	/**
+	 * The default maximum message size when deserializing.
+	 * @since 5.1.3
+	 */
+	public static final int DEFAULT_MAX_MESSAGE_SIZE = 2048;
+
+	protected int maxMessageSize = DEFAULT_MAX_MESSAGE_SIZE;
```

Comment:
Isn't Sonar going to object that we continue to keep it as a `protected`?
Although I guess the `private` and `protected` getter is going to be as a breaking change in the point release...

Reply:
Doh - I meant it to be public - this explains why I couldn't get javadocs `@value` to work.

Fixing...

Reply:
OK; will fix.